packages/pn-personagiuridica-webapp/public/locales/it/notifiche.json

```diff
@@ -57,11 +57,9 @@
     "add_domicile_1": "Aggiungi PEC",
     "add_domicile_2": "Aggiungi cellulare",
     "add_domicile_3": "Aggiungi email",
-    "add_domicile_4": "Attiva il servizio",
     "domicile_1": "Hai un indirizzo PEC? Aggiungilo e non riceverai più le raccomandate cartacee.",
     "domicile_2": "Aggiungi il tuo numero di cellulare: ti invieremo un messaggio ogni volta che ricevi una notifica.",
     "domicile_3": "Aggiungi il tuo indirizzo email: ti invieremo un messaggio ogni volta che ricevi una notifica.",
```

Comment:
the strings domicile_# and add_domicile_# seems to not be used in the app after these changes, i think they should be removed